release/prow.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ "$AWS_ROLE_ARN" == "" ]; then
+    echo "Empty AWS_ROLE_ARN"
+    exit 1
+fi
+
+if [ "$ARTIFACT_DEPLOYMENT_ROLE_ARN" == "" ]; then
+    echo "Empty ARTIFACT_DEPLOYMENT_ROLE_ARN"
+    exit 1
+fi
+
+BASE_DIRECTORY=$(git rev-parse --show-toplevel)
+cd ${BASE_DIRECTORY}
+
+cat << EOF > awscliconfig
+[default]
+output=json
+region=${AWS_REGION:-${AWS_DEFAULT_REGION:-us-west-2}}
+role_arn=$AWS_ROLE_ARN
+web_identity_token_file=/var/run/secrets/eks.amazonaws.com/serviceaccount/token
+
+[profile release-account]
+role_arn=$ARTIFACT_DEPLOYMENT_ROLE_ARN
+region=${AWS_REGION:-${AWS_DEFAULT_REGION:-us-east-1}}
```

Comment:
Using AWS_REGION will be a problem here as the Pod already injects the region. Maybe we can add an `ARTIFACTS_AWS_REGION` env var to our prowjob and use that here.